### PR TITLE
update editor community plugins

### LINKED ISSUE #2660: Add Nova editor extension
https://www.prisma.io/docs/guides/development-environment/editor-setup
https://github.com/prisma/prisma/issues/7223#issuecomment-1004760770

### PR DESCRIPTION
**Pull request by Jolg42**
Small update and cleanup.

closes https://github.com/prisma/docs/issues/2660

### CODE REVIEW

File: content/300-guides/125-development-environment/100-editor-setup.mdx [hunk 1]
```diff
@@ -23,12 +23,19 @@ You can install the [Prisma VS Code extension](https://marketplace.visualstudio.
 
 ### Emacs
 
-For Emacs you can use [emacs-prisma-mode](https://github.com/pimeys/emacs-prisma-mode).
+[emacs-prisma-mode](https://github.com/pimeys/emacs-prisma-mode).
```
Comment (Jolg42) [inline content/300-guides/125-development-environment/100-editor-setup.mdx]:
```suggestion
[emacs-prisma-mode](https://github.com/pimeys/emacs-prisma-mode) provides syntax highlighting of the Prisma Schema Language and use the Prisma Language Server.
```

File: content/300-guides/125-development-environment/100-editor-setup.mdx [hunk 2]
```diff
@@ -23,12 +23,23 @@ You can install the [Prisma VS Code extension](https://marketplace.visualstudio.
...
 
 ### Jetbrains IDE
 
-[Jetbrains](https://plugins.jetbrains.com/plugin/14240-prisma) offers an extension to support the Prisma Schema Language.
+[Prisma](https://plugins.jetbrains.com/plugin/14240-prisma) supports syntax highlighting of the Prisma Schema Language.
+[Prisma Support](https://plugins.jetbrains.com/plugin/14282-prisma-support) supports syntax highlighting of the Prisma Schema Language.
+
+### Sublime Text
+
+[sublimeText3](https://plugins.jetbrains.com/plugin/14240-prisma) supports syntax highlighting of the Prisma Schema Language.
```
Comment (Jolg42) [inline content/300-guides/125-development-environment/100-editor-setup.mdx]:
```suggestion
[sublimeText3](https://github.com/prisma/sublimeText3) supports syntax highlighting of the Prisma Schema Language.
```

**Comment by Jolg42** [comment]
Should be ready to merge now :)

File: content/300-guides/125-development-environment/100-editor-setup.mdx [hunk 3]
```diff
@@ -23,12 +23,25 @@ You can install the [Prisma VS Code extension](https://marketplace.visualstudio.
 
 ### Emacs
 
-For Emacs you can use [emacs-prisma-mode](https://github.com/pimeys/emacs-prisma-mode).
+[emacs-prisma-mode](https://github.com/pimeys/emacs-prisma-mode) provides syntax highlighting of the Prisma Schema Language and use the Prisma Language Server.
```
Comment (janpio) [inline content/300-guides/125-development-environment/100-editor-setup.mdx]:
```suggestion
[emacs-prisma-mode](https://github.com/pimeys/emacs-prisma-mode) provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language Server.
```

File: content/300-guides/125-development-environment/100-editor-setup.mdx [hunk 4]
```diff
@@ -23,12 +23,25 @@ You can install the [Prisma VS Code extension](https://marketplace.visualstudio.
...
+[Prisma](https://plugins.jetbrains.com/plugin/14240-prisma) provides syntax highlighting of the Prisma Schema Language.
+[Prisma Support](https://plugins.jetbrains.com/plugin/14282-prisma-support) provides syntax highlighting of the Prisma Schema Language.
+
+### Sublime Text
+
+[sublimeText3](https://github.com/prisma/sublimeText3) provides syntax highlighting of the Prisma Schema Language.
+
+### nova
+
+[Prisma](https://extensions.panic.com/extensions/robb-j/robb-j.Prisma/) provides syntax highlighting of the Prisma Schema Language and use the Prisma Language Server.
```
Comment (janpio) [inline content/300-guides/125-development-environment/100-editor-setup.mdx]:
```suggestion
[Prisma](https://extensions.panic.com/extensions/robb-j/robb-j.Prisma/) provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language Server.
```

**Review by tanberry — APPROVED** [review]
LGTM, thanks!

**Pull request merged**